http-client/src/main/java/io/micronaut/http/client/netty/ConnectionManager.java

```diff
@@ -209,10 +212,13 @@ public class ConnectionManager {
 
     final void refresh() {
         SslContext oldSslContext = sslContext;
+        SslContext oldWebsocketSslContext = websocketSslContext;
         if (configuration.getSslConfiguration().isEnabled()) {
             sslContext = nettyClientSslBuilder.build(configuration.getSslConfiguration(), httpVersion);
+            websocketSslContext = nettyClientSslBuilder.build(configuration.getSslConfiguration(), HttpVersionSelection.forWebsocket());
```

Comment:
can we make this lazy, id rather not have it sitting around unless necessary

Reply:
Check [31f68f8](https://github.com/micronaut-projects/micronaut-core/pull/10754/commits/31f68f8dcee8b32ac0dbe98cd103fa718cafc1e5) and see what you think. It will now create  one per WebSocket connection. The context that gets created does not require manual calls to release.

Is this too far in the opposite direction? Is the SSLContext building too expensive to be doing it for each connection?